### PR TITLE
added necessary open source files

### PR DESCRIPTION
**Pull request by silasburger**
This pull request includes a

- [ ] Bug fix
- [x] New feature
- [x] Translation

The following changes were made

- I added the following: 
- .github/ISSUE_TEMPLATE.md
- .github/PULL_REQUEST_TEMPLATE.md
- LICENSE.md
- CODE_OF_CONDUCT.md
- CONTRIBUTING.md

If this is related to an existing ticket, include a link to it as well.

https://github.com/rithmschool/numbers_api/issues/4

### CODE REVIEW

**Comment by mmmaaatttttt** [comment]
@maxnawa31 want to give this one a look? not sure if you two ever connected on this pr -- if not let me know, we should standardize on where communication happens!

**Comment by maxnawa31** [comment]
yea sure thing, silas slacked me a couple days ago about this PR, i just haven't had time to dive into it. I'll look into it today

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+-->
+**Problem description**:
+
+**Suggested solution**:
+
+**Relevant code snippet**:
+
+```javascript
+```
+
```
Comment (maxnawa31) [inline .github/ISSUE_TEMPLATE.md:14]: Just an idea, but what do you think about adding the following information categories?

1) Steps to Reproduce
2)The current behavior
3) The expected behavior

I stole these from the React open source issue template, linked here if you want to take a look [https://github.com/facebook/react/blob/master/.github/ISSUE_TEMPLATE/bug_report.md](url)

**Comment by silasburger** [comment]
Awesome

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
+available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
+
+[homepage]: https://www.contributor-covenant.org
+
+For answers to common questions about this code of conduct, see
+https://www.contributor-covenant.org/faq
```
Comment (maxnawa31) [inline CODE_OF_CONDUCT.md:77]: 👍

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Contributing
+
+Thanks for being willing to contribute!
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a build.
+2. Use the PR template to provide important information about your work.
+3. Your pull request will be reviewed in two separate stages. The first stage requires that an authorized contributor 
+signs off on the changes. The second stage required the head reviewer to take a look and make the appropriate changes.
```
Comment (maxnawa31) [inline CONTRIBUTING.md:10]: quik question but do we want to add tests as a step for the PR process?

File: LICENSE.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
Comment (maxnawa31) [inline LICENSE.md:20]: 👍

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+- [ ] New feature
+- [ ] Translation
+
+The following changes were made
+
+-
+-
+-
+
+If this is related to an existing ticket, include a link to it as well.
```
Comment (maxnawa31) [inline .github/PULL_REQUEST_TEMPLATE.md:13]: 👍

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (mmmaaatttttt) [reply .github/ISSUE_TEMPLATE.md:14]: +1 this @maxnawa31

File: CONTRIBUTING.md (hunk 3 above)
Comment (mmmaaatttttt) [reply CONTRIBUTING.md:10]: +1 @maxnawa31

**Comment by mmmaaatttttt** [comment]
@silasburger @maxnawa31 where are we with this one? I know Max made some good suggestions, who wants to own those updates?

**Comment by silasburger** [comment]
Hi Matt,

I can add the updates within the next couple of days.

Best,
Silas

File: .github/ISSUE_TEMPLATE.md [hunk 6]
```diff
@@ -5,6 +5,12 @@ Fill out this template with information that is relevant to the issue and if nec
 -->
 **Problem description**:
 
+**Steps to Reproduce**:
+
+**The current behavior**:
+
+**The expectec behavior**:
```
Comment (maxnawa31) [inline .github/ISSUE_TEMPLATE.md:12]: small typo here

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+
+Thanks for being willing to contribute!
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a build.
+2. Use the PR template to provide important information about your work.
+3. When adding new code of fixing a bug make sure to add tests to avoid future regressions.
+4. Your pull request will be reviewed in two separate stages. The first stage requires that an authorized contributor 
+signs off on the changes. The second stage required the head reviewer to take a look and make the appropriate changes.
```
Comment (maxnawa31) [inline CONTRIBUTING.md:11]: Looks good

**Comment by silasburger** [comment]
Okay sounds good

**Comment by mmmaaatttttt** [comment]
@silasburger you're one character away from a merged PR!

**Comment by silasburger** [comment]
I finally fixed the typo!!

**Pull request merged**